backend/README.md

```diff
@@ -1,11 +1,25 @@
-# backend
+# EmbassyOS Backend
 - Requirements:
     - [Install Rust](https://rustup.rs)
     - Recommended: [rust-analyzer](https://rust-analyzer.github.io/)
     - [Docker](https://docs.docker.com/get-docker/)
     - [Rust ARM64 Build Container](https://github.com/Start9Labs/rust-arm-builder)
-- Installation: 
-    - follow the setup instructions [here](backend/README.md)
 - Scripts (run withing the `./backend` directory)
     - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
     - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
+- A Linux computer or VM
+
+## Structure
+
+The EmbassyOS backend is broken up into 4 different binaries:
+
+- embassyd: This is the main workhorse of EmbassyOS any new functionality you want will likely go here
```

Comment:
`embassyd: This is the main workhorse of EmbassyOS - any new functionality you want will likely go here`